### PR TITLE
feat(rff-21): create table component with the feature

### PR DESCRIPTION
**Pull request by jisung-lee7**

### CODE REVIEW

**Review by jisung-lee7 — COMMENTED** [review]
test

**Comment by jisung-lee7** [comment]
DONE ! but jiyeol, it just works. 
I’m still not entirely sure in which situations it’s better to add more state or keep it minimal. 
For now, my approach 'has been to avoid creating additional state as much as possible'. 
If any optimizations are better, I’d appreciate it if you could leave a comment anytime.

File: src/modules/github/components/bulk-pull-reviews/bulk-pull-reviews.tsx [hunk 1]
```diff
@@ -25,22 +32,95 @@ const Repositories = ({ repos }: { repos: TRepo[] | undefined }) => {
...
+    repos.length > 0 &&
+    repos.filter((repo) => repo.isOpen).length > 0 &&
+    repos
+      .filter((repo) => repo.isOpen)
+      .every(
+        (repo) =>
+          repo.pulls &&
+          repo.pulls.length > 0 &&
+          repo.pulls?.every((pull) => pull.checked)
+      )
```
Comment (jiyeol-lee) [inline src/modules/github/components/bulk-pull-reviews/bulk-pull-reviews.tsx:50]: Can you detach this into a function and write a test code?

File: src/modules/github/components/bulk-pull-reviews/bulk-pull-reviews.tsx [hunk 2]
```diff
@@ -25,22 +32,95 @@ const Repositories = ({ repos }: { repos: TRepo[] | undefined }) => {
...
+                          )
+                        )
+                      }}
+                      disabled={!hasRepoChild}
+                    />
+                  )
+                },
+                {
+                  key: 'header-cell-1',
+                  children: 'Repository'
```
Comment (jiyeol-lee) [inline src/modules/github/components/bulk-pull-reviews/bulk-pull-reviews.tsx:78]: What is this `children: 'Repository'`?

File: src/modules/github/components/bulk-pull-reviews/bulk-pull-reviews.tsx [hunk 3]
```diff
@@ -25,22 +32,95 @@ const Repositories = ({ repos }: { repos: TRepo[] | undefined }) => {
...
-              installationId={repo.installationId}
-              owner={repo.owner}
-              repo={repo.name}
-              repoUrl={repo.url}
-              pulls={repo.pulls}
-              isRepoOpen={repo.isOpen}
-            />
-          ))
-        : null}
+      {repos ? (
```
Comment (jiyeol-lee) [inline src/modules/github/components/bulk-pull-reviews/bulk-pull-reviews.tsx:54]: What about just do
```typescript
if (!repos) {
  return null
}

return (
  <Table
    blah blah blah
  />
```

File: src/modules/github/components/pull-list-by-repo/pull-list-by-repo.tsx [hunk 4]
```diff
@@ -34,14 +35,16 @@ export const PullListByRepo = ({
...
+    /* toggleRepoCheckStatus, */ togglePullCheckStatus,
+    toggleRepoOpenStatus
+  } = useUpdateRepoOrPull()
   // TODO: handle error case
-  const { isPending, isLoading } = useFetchPulls(owner, repo, isRepoOpen)
-  const hasChild = !!pulls && pulls.length > 0
-  const isRepoChecked = !!pulls
-    ? pulls.length > 0 && pulls.every((pull) => pull.checked)
-    : false
+  const { /* isPending, */ isLoading } = useFetchPulls(owner, repo, isRepoOpen)
```
Comment (jiyeol-lee) [inline src/modules/github/components/pull-list-by-repo/pull-list-by-repo.tsx:43]: Can you remove all of these comments? It's very difficult to review. If you put it temporary, backup somehow on your local machine using stash or in a different branch.

Comment (jisung-lee7) [reply src/modules/github/components/pull-list-by-repo/pull-list-by-repo.tsx:43]: Ah sorry, I forgot it

**Review by jiyeol-lee — APPROVED** [review]
Looks good to me

**Comment by jisung-lee7** [comment]
> Looks good to me

Awesome!

**Pull request merged**